entproto/adapter.go

```diff
@@ -510,9 +515,20 @@ func extractProtoTypeDetails(f *gen.Field) (fieldType, error) {
 	}, nil
 }
 
+func extractJSONDetails(f *gen.Field) (fieldType, error) {
+	if f.Type.Ident == "[]string" {
```

Comment:
We also have []float64 and []int

Reply:
Yep, this PR adds support for field.Strings, the rest will come after